Tests/MongoSwiftTests/DocumentTests.swift

```diff
@@ -55,6 +55,10 @@ extension Document {
             return DocElem(key: kvp.key, value: kvp.value, type: kvp.value.bsonType)
         }
     }
+
+    internal var data: MutableBSONPointer {
```

Comment:
I've moved this into the tests because various tests in this file checking copy on write behavior were using it. but it is no longer used anywhere in the driver.